spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkReadConf.java

```diff
@@ -353,4 +354,19 @@ private boolean executorCacheLocalityEnabledInternal() {
         .defaultValue(SparkSQLProperties.EXECUTOR_CACHE_LOCALITY_ENABLED_DEFAULT)
         .parse();
   }
+
+  private ParquetReaderType parquetReaderType() {
+    return confParser
+        .enumConf(ParquetReaderType::valueOf)
+        .sessionConf(SparkSQLProperties.PARQUET_READER_TYPE)
+        .defaultValue(SparkSQLProperties.PARQUET_READER_TYPE_DEFAULT)
+        .parse();
+  }
+
+  public BatchReadConf batchReadConf() {
+    int parquetBatchSize = parquetBatchSize();
+    int orcBatchSize = orcBatchSize();
+    Preconditions.checkArgument(parquetBatchSize > 1 || orcBatchSize > 1, "Batch size must be > 1");
```

Comment:
Why `||`?

Reply:
I didn't realize that both `parquetBatchSize` and `orcBatchSize` have default size. I will change to `&&`